frontend/src/v5/ui/routes/dashboard/projects/projectsList/createProjectModal/createProjectModal.component.tsx

```diff
@@ -107,7 +100,7 @@ export const CreateProjectForm = ({ open, onClickClose }: ICreateProject) => {
 			onClickClose={onClickClose}
 			onSubmit={handleSubmit(onSubmit)}
 			confirmLabel={formatMessage({ id: 'projects.creation.ok', defaultMessage: 'Create Project' })}
-			isValid={formState.isValid && !hasUnexpectedError}
+			isValid={formState.isValid}
```

Comment:
what happens now id there is an unexpected error? this form becames invalid?

Reply:
Now an unexpected error does not affect the validity of the form, i.e. if the form is valid per se, but there is an unexpected error, the form remains valid. I did this here and in another file to match the rest of the forms which are still consider valid when an unexpected error is caught